packages/jest-repl/src/cli/repl.ts

```diff
@@ -5,30 +5,38 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @flow
  */
 
-import type {GlobalConfig, ProjectConfig} from 'types/Config';
-
-declare var jestGlobalConfig: GlobalConfig;
-declare var jestProjectConfig: ProjectConfig;
-declare var jest: Object;
+declare var jestGlobalConfig: Config.GlobalConfig;
+declare var jestProjectConfig: Config.ProjectConfig;
 
 import path from 'path';
 import repl from 'repl';
 import vm from 'vm';
+import {Transformer} from '@jest/transform';
+import {Config} from '@jest/types';
 
-let transformer;
+let transformer: Transformer;
 
-const evalCommand = (cmd, context, filename, callback, config) => {
+const evalCommand: repl.REPLEval = (
+  cmd: string,
+  _context: any,
+  _filename: string,
+  callback: (e: Error | null, result?: any) => void,
+  _config?: any,
+) => {
   let result;
   try {
     if (transformer) {
-      cmd = transformer.process(
+      const transformResult = transformer.process(
         cmd,
         jestGlobalConfig.replname || 'jest.js',
```

Comment:
this seems weird to me, `GlobalConfig` is a type. You can do `import {Defaults} from 'jest-config` instead? I'm not sure how this works, tbh

Reply:
Ye me too. It think someone else with more knowledge chipping in would be appreciated. Anyways, I’ll investigate further tomorrow on my end.

Reply:
Dig into it (#1195) - `jestGlobalConfig` was set as a global by Jest. I don't think that's true anymore though (or, it is inside of tests, but this isn't inside of a test, I think)

Reply:
Dug a bit more, and this does actually work. This `repl` file is required inside of the jest context, so it does have those globals, set here: https://github.com/facebook/jest/blob/4023b8c8e1432ed788d33df48b69794c422e4880/packages/jest-runtime/src/cli/index.ts#L86-L92